doozer/doozerlib/cli/images_streams.py

```diff
@@ -1267,9 +1274,11 @@ def check_if_upstream_image_exists(upstream_image):
   introduced with a separate PR opened by the component team. Once the repository is aligned,
   this PR will be closed automatically.
 - Patch-manager or those with sufficient privileges within this repository may add
-  any required labels to ensure the PR merges once tests are passing. Downstream builds
-  are *already* being built with these changes. Merging this PR only improves the fidelity
-  of our CI.
+  any required labels to ensure the PR merges once tests are passing. In cases where ART config is
+  canonical, downstream builds are *already* being built with these changes, and merging this PR
```

Comment:
Should we mention the expected behavior of our canonical feature? 
preGA: usually canonical 
postGA: not canonical